src/utils/generalUtils.ts

```diff
@@ -46,4 +50,8 @@ export namespace GeneralUtils {
         }
         return fn(e.result);
     }
+
+    export function getApiCenterWorkspacePath(): string {
+        return path.join(os.homedir(), apiCenterFolder);

```

Comment:
where to ensure this folder exist?

Reply:
This function is to get the path. When exporting the rules, will create folder if it does not exist.